src/deploy-config/base-config.ts

```diff
@@ -0,0 +1,191 @@
+  PROPOSITION_THRESHOLD: 50,
+  VOTING_DURATION_BLOCKS: ONE_DAY_BLOCKS * 2,
+  VOTE_DIFFERENTIAL: 50,
+  MINIMUM_QUORUM: 100,
+};
+
+const config = {
+  // Common schedule parameters.
+  EPOCH_LENGTH,
+  BLACKOUT_WINDOW,
+  REWARDS_DISTRIBUTION_LENGTH,
+
+  // DYDX token parameters.
+  MINT_MAX_PERCENT: 2,
+
+  // Governance parameters.
+  VOTING_DELAY_BLOCKS: 6570, // One day, assuming average block time of 13s.
+
+  // Treasury parameters.
+  REWARDS_TREASURY_VESTING_AMOUNT: new BNJS(toWad(450_000_000))
+    .minus(REWARDS_TREASURY_FRONTLOADED_FUNDS)
+    .toFixed(),
+  COMMUNITY_TREASURY_VESTING_AMOUNT: toWad(50_000_000),
+
+  // Timelock parameters.
+  LONG_TIMELOCK_CONFIG,
+  SHORT_TIMELOCK_CONFIG,
+  MERKLE_PAUSER_TIMELOCK_CONFIG,
+
+  // Merkle Distributor.
+  MERKLE_DISTRIBUTOR_WAITING_PERIOD,
+
+  // Safety Module.
+  SM_REWARDS_PER_SECOND: STAKING_REWARDS_PER_SECOND,
+
+  // Liquidity Staking.
+  LS_MIN_BLACKOUT_LENGTH: 3 * ONE_DAY_SECONDS,
+  LS_MAX_EPOCH_LENGTH: 92 * ONE_DAY_SECONDS,
+  LS_REWARDS_PER_SECOND: STAKING_REWARDS_PER_SECOND,
+  getLiquidityStakingMinEpochLength() {
+    return this.LS_MIN_BLACKOUT_LENGTH * 2;
+  },
+  getLiquidityStakingMaxBlackoutLength() {
+    return this.LS_MAX_EPOCH_LENGTH / 2;
+  },
+
+  // Treasuries.
+  REWARDS_TREASURY_FRONTLOADED_FUNDS,
+
+  // Initial token allocations.
+  TOKEN_ALLOCATIONS: {
+    TEST_TOKENS_1: {
+      ADDRESS: '0xeb74327ddd3f0d359321f06d84a9d3871b4d96a4',
+      AMOUNT: toWad(2),
+    },
+    TEST_TOKENS_2: {
+      ADDRESS: '0xb03414a51a625e8ce16d284e34941ba66c5683c9',
+      AMOUNT: toWad(2),
+    },
+    TEST_TOKENS_3: {
+      ADDRESS: '0x1eec5afab429859c46db6552bc973bdd525fd7b1',
+      AMOUNT: toWad(2),
+    },
+    TEST_TOKENS_4: {
+      ADDRESS: '0x69112552fac655bb76a3e0ee7779843451db02b6',
+      AMOUNT: toWad(2),
+    },
+    DYDX_FOUNDATION: {
+      ADDRESS: '0xb4fbF1Cd41BB174ABeFf6001B85490b58b117B22',
+      AMOUNT: toWad(293_355_248.288681),
+    },
+    DYDX_TRADING: {
+      ADDRESS: '0xf95746B2c3D120B78Fd1Cb3f9954CB451c2163E4',
+      AMOUNT: toWad(115_941_170.358637),
+    },
+    DYDX_LLC: {
+      ADDRESS: '0xCc9507708a918b1d44Cf63FaB4E7B98b7517060f',
+      AMOUNT: toWad(90_703_573.352682),
+    },
+  },
+
+  // Addresses which were used on mainnet for token custody testing.
+  TOKEN_TEST_ADDRESSES: [
+    '0xeb74327ddd3f0d359321f06d84a9d3871b4d96a4',
+    '0xb03414a51a625e8ce16d284e34941ba66c5683c9',
+    '0x1eec5afab429859c46db6552bc973bdd525fd7b1',
+    '0x69112552fac655bb76a3e0ee7779843451db02b6',
+    '0x688134aefae00632d36754ca9f085bd25072e2f0',
+    '0xb50309682298fecf08ea07792154fa26f05fd7e5',
+    '0x49fae0a92272ec68583eb009f4c40bcf91b9010f',
+    '0xad2955994fb189117478eec154153ba7d0f835ca',
+  ],
+
+  // Safety Module recovery.
+  //
+  // Compensation amount is 10% of what was initially staked and stuck in the Safey Module.
+  SM_RECOVERY_COMPENSATION_AMOUNT: '15745887438533773204745', // About 15746 DYDX.
```

Comment:
Super small nit: Include comment on how this was calculated + what date we snapshotted the funds at?`157,458.87438533773204764 * 0.1 = 15,745.8874385338` (using [safety module balance on Etherscan]( https://etherscan.io/token/0x92d6c1e31e14520e676a687f0a93788b716beff5?a=0x65f7BA4Ec257AF7c55fd5854E5f6356bBd0fb8EC)).

Reply:
Updated comments

Reply:
Updated comments